Assets/Scripts/Game/GameWaveManager.cs

```diff
@@ -53,9 +54,15 @@ private List<GameWaveDto> ReadGameData(GameDifficulty difficulty)
             return gameWaves;
         }
 
-        private static GameWaveDto ParseWave(int waveNumber, string waveString)
+        private GameWaveDto ParseWave(int waveNumber, string waveString)
         {
             var wave = waveString.Split(',');
+            var isOilAffectLight = int.Parse(wave[6]) != 0;
+            var waveDuration = float.Parse(wave[7], CultureInfo.InvariantCulture);
+            if (!isOilAffectLight)
```

Comment:
That is a bit sloppy since it won't work if we disable/enabled lights in waves but that is ok for now.
If needed, I may change this to count only until the first switch off.

Reply:
Its' completely ok for now I suppose